frontend/src/scenes/insights/PersonModal.js

```diff
@@ -33,7 +34,14 @@ export function PersonModal({ visible, view }) {
                 <div
                     style={{ marginBottom: 16, display: 'flex', alignItems: 'center', justifyContent: 'space-between' }}
                 >
-                    Found {people.count === 99 ? '99+' : people.count} {people.count === 1 ? 'user' : 'users'}
+                    <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-start' }}>
+                        Found {people.count === 99 ? '99+' : people.count} {people.count === 1 ? 'user' : 'users'}
+                        {(view === ViewType.TRENDS || view === ViewType.STICKINESS) && (
```

Comment:
Q: Why not in other views? I can see this being really useful in retention.

Reply:
definitely should be in other views but this PR was getting heavy so i cut it to trends/stickiness first